tests/e2e/addremoverereplacevmtodefaulttenant_test.go

```diff
@@ -0,0 +1,67 @@
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Pls. change file name to a small one like defaultvmgroup.go, #1262 is also adding tests for the default tenant and may make it into this file vs. a file for each test.